src/Reference/ReferenceMap.php

```diff
@@ -19,6 +19,7 @@
  */
 class ReferenceMap
 {
+
```

Comment:
My bad. One of my PSR compatible code style rules that I use for my projects. Amended the commit.